.dockerignore

```diff
@@ -0,0 +1,15 @@
+.git
+*.md
+
+.idea
+.idea/*
+
+/var/
+/vendor/
+
+###< symfony/framework-bundle ###
+###> symfony/phpunit-bridge ###
+/phpunit.xml.dist
+/phpunit.xml
+phpunit.xml.dist
```

Comment:
Remove duplications

Reply:
this also updated and merged with its base branch, but its still here?